extensions/amp-recaptcha-input/0.1/test/test-amp-recaptcha-service.js

```diff
@@ -144,6 +144,31 @@ describes.realWin('amp-recaptcha-service', {
         });
   });
 
+  it('should postmessage to the specificed ' +
+    'recaptcha frame origin on execute', () => {
+
+    expect(recaptchaService.registeredElementCount_).to.be.equal(0);
+
+    const postMessageStub = sandbox.stub(
+        recaptchaService,
+        'postMessageToIframe_'
+    );
+
+    return recaptchaService
+        .register(fakeSitekey).then(() => {
+          expect(recaptchaService.unlisteners_.length).to.be.equal(3);
```

Comment:
why 3? doesn't seem to be intuitive. 

also, can we avoid looking into this private member? we don't really care about the number of unlisteners, do we?